bears/vcs/git/GitCommitBear.py

```diff
@@ -104,8 +104,19 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
 
         stdout = stdout.rstrip('\n')
         pos = stdout.find('\n')
+        count = 0
         shortlog = stdout[:pos] if pos != -1 else stdout
-        body = stdout[pos+1:] if pos != -1 else ''
+        pos = pos + 1
+
+        while (stdout[pos+1] == '\n'):
+            count++
+
+        if count == 0:
+            body = stdout[pos+1:] if pos != -1 else ''
+        else
+            yield Result(self,
```

Comment:
This will result in runtime exceptions when there are more than one `\n`, as you don't assign `body` then which is used later. Move `body = ...` out of the if